src/pages/search.jsx

```diff
@@ -1,21 +1,31 @@
-import { Link, Outlet, useLocation } from 'react-router-dom';
+import { useContext, useEffect, useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
 
 import ButtonRandom from '../components/button-random';
+import { countryList } from '../components/country-list';
+import { ChoicesContext } from '../contexts/choices-context';
 import './search.scss';
 
-const placeholderList = [
-  { language: 'Français', placeholderTraduc: 'Ecrivez votre recherche en français' },
-  { language: 'English', placeholderTraduc: 'Write your search in english' },
-  { language: 'Deutsch', placeholderTraduc: 'Schreiben Sie Ihre Suche auf Deutsch' },
-  { language: '中文', placeholderTraduc: '用中文写下您的搜索' },
-  { language: 'Español', placeholderTraduc: 'Escriba su búsqueda en español' },
-];
-
 export default function Search() {
+  const [listeLanguageActive, setListeLanguageActive] = useState(false);
```

Comment:
En anglais `list`
C'est une bonne pratique de nommer le début des booleans par `is`, `has`, `should`, `can`, ... -> `isActiveLanguageList`, `shouldShowList`, ...